Cannot select "Font Family" on Windows builds

**Synfig version & platform**:
1.5.0 Artifact build of 5 October 4699e374 (got from a pull request)

**Issue description**:
1. Create a new animation
2. Insert a `Text Layer` on Canvas
3. Change the `Font Family` value in the `Parameters Panel`

The dropdown menu is greyed out and it's not possible to select any fonts.
I could not use it the "old" way i.e. by copying the font file name in the `Font Family` box.

@ice0 

I tried to do test-build on my local machine using your given commands but I got some errors.
-- Configuring done
-- Generating done
-- Build files have been written to: C:/synfig/test-build1
make[2]: *** No rule to make target 'output/lib/libsynfig.dll.a', needed by 'output/lib/synfig/modules/liblyr_freetype.dll'.  Stop.
make[1]: *** [CMakeFiles/Makefile2:1051: synfig-core/src/modules/lyr_freetype/CMakeFiles/lyr_freetype.dir/all] Error 2
make: *** [Makefile:172: all] Error 2

And the only error or flaw I found in your given output is DWITH_FONTCONFIG which is present in the lyr_freetype but not in the  widget_font. 

Thanks for help :)

> make: *** [Makefile:172: all] Error 2

Just use Ninja in a different folder as I wrote earlier.

> And the only error or flaw I found in your given output is DWITH_FONTCONFIG which is present in the lyr_freetype but not in the widget_font.

Yes, this is the reason, why widget not working.

--- a/synfig-core/src/modules/lyr_freetype/lyr_freetype.cpp
+++ b/synfig-core/src/modules/lyr_freetype/lyr_freetype.cpp
@@ -283,7 +283,11 @@ TextLine::clear_and_free()
 
 static bool
 has_valid_font_extension(const std::string &filename) {
-	return std::find(known_font_extensions.begin(), known_font_extensions.end(), filename) != known_font_extensions.end();
+	auto pos = filename.find_last_of(".");
+	if (pos == filename.npos)
+		return false;
+	std::string extension = filename.substr(pos);
+	return std::find(known_font_extensions.begin(), known_font_extensions.end(), extension) != known_font_extensions.end();
 }
 
 

